OpenRA.Game/Settings.cs

```diff
@@ -72,7 +72,7 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
-		public string TimestampFormat = "HH:mm";
+		public string TimestampFormat = "dd/MM - HH:mm:ss";
```

Comment:
https://msdn.microsoft.com/en-us/library/az4se3k1%28v=vs.110%29.aspx#Roundtrip
Looks like it's "o"